lib/elixir/lib/enum.ex

```diff
@@ -1890,9 +1888,19 @@ defmodule Enum do
       iex> Enum.slice(1..10, 5..20)
       [6, 7, 8, 9, 10]
 
+      # last five elements (negative positions)
+      iex> Enum.slice(1..30, -5..-1)
+      [26, 27, 28, 29, 30]
+
+      # last five elements (mixed positive and negative positions)
+      iex> Enum.slice(1..30, 25..-1)
+      [26, 27, 28, 29, 30]
+
+      # out of bounds
       iex> Enum.slice(1..10, 11..20)
       []
 
+      # range.first is greater than range.last
```

Comment:
the comment precedes the example.
It looks fine to me.
you can see docs for `Code.get_docs/2`

these are all the places where we use them.

```sh
r$ ag "#.*\n +iex>"
lib/iex/lib/iex/helpers.ex
661:      #PID<0.21.32>
662:      iex> pid(0, 64, 2048)

lib/elixir/lib/stream.ex
1098:      # Although not necessary, let's seed the random algorithm
1099:      iex> :rand.seed(:exsplus, {1, 2, 3})

lib/elixir/lib/protocol.ex
140:      # Get Elixir's ebin and retrieve all protocols
141:      iex> path = :code.lib_dir(:elixir, :ebin)
170:      # Get Elixir's ebin and retrieve all protocols
171:      iex> path = :code.lib_dir(:elixir, :ebin)

lib/elixir/lib/enum.ex
1592:      # Although not necessary, let's seed the random algorithm
1593:      iex> :rand.seed(:exsplus, {101, 102, 103})
1854:      # Although not necessary, let's seed the random algorithm
1855:      iex> :rand.seed(:exsplus, {1, 2, 3})
2331:      # Although not necessary, let's seed the random algorithm
2332:      iex> :rand.seed(:exsplus, {1, 2, 3})

lib/elixir/lib/calendar.ex
357:      # Time with microseconds and their precision
358:      iex> Time.new(23, 59, 60, {10_000, 2})

lib/elixir/lib/code.ex
600:      # Get the module documentation
601:      iex> {_line, text} = Code.get_docs(Atom, :moduledoc)
605:      # Module doesn't exist
606:      iex> Code.get_docs(ModuleNotGood, :all)

lib/elixir/lib/kernel/special_forms.ex
1231:      # A list generator:
1232:      iex> for n <- [1, 2, 3, 4], do: n * 2
1235:      # A comprehension with two generators
1236:      iex> for x <- [1, 2], y <- [2, 3], do: x*y
1241:      # A comprehension with a generator and a filter
1242:      iex> for n <- [1, 2, 3, 4, 5, 6], rem(n, 2) == 0, do: n
```